elasticsearch/README.md

```diff
@@ -39,10 +39,10 @@ This helm chart is a lightweight way to configure and run our official [Elastics
 | -------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------- |
 | `clusterName`              | This will be used as the Elasticsearch [cluster.name](https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster.name.html) and should be unique per cluster in the namespace                                                                                                                                 | `elasticsearch`                                                                                                           |
 | `nodeGroup`                | This is the name that will be used for each group of nodes in the cluster. The name will be `clusterName-nodeGroup-X`                                                                                                                                                                                                      | `master`                                                                                                                  |
-| `masterService`            | Optional. The service name used for [discovery.zen.ping.unicast.hosts](https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-discovery-zen.html#unicast) to connect to the masters. You only need to set this if your master `nodeGroup` is set to something other than `master`                         | ``                                                                                                                        |
+| `masterService`            | Optional. The service name used for [discovery.zen.ping.unicast.hosts] to connect to the masters. You only need to set this if your master `nodeGroup` is set to something other than `master`. Deprecated but currently supported in versions >= 7, replaced by [discovery.seed_hosts].                                   | ``                                                                                                                        |
```

Comment:
> Deprecated but currently supported

This is a bit confusing since it sounds like the `masterService` value is deprecated. In reality it is just an implementation detail. Instead we should update the chart to use `discovery.seed_hosts` when the version is 7 or greater. Deprecated makes it sound like the user should stop using this value which is not the case here.

Reply:
Instead I think it is better to just update the wording to remove any implementation details and link to `#clustering-and-node-discovery` which explains it in more details. 

So something like:

Optional. The service name used to connect to the masters. You only need to set this if your master `nodeGroup` is set to something other than `master`. See #clustering-and-node-discovery for more information.

Reply:
Updated with your suggested description.